securedrop/i18n_tool.py

```diff
@@ -389,7 +389,7 @@ def translators(
 
         log_command.extend([args.target, "--", path])
 
-        log_lines = subprocess.check_output(log_command, encoding="utf-8").strip().splitlines()
+        log_lines = subprocess.check_output(log_command, encoding="utf-8").strip().split("\n")
```

Comment:
Can you leave a comment here explaining why splitlines isn't used?

Reply:
It didn't occur to me to comment on the reverted code, but of course this (along with the new test stanza) will help guard against another regression.  Added in c761bb790ae4793add4ad7ad0aa6328f02206ae7.